.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 services:
   - postgresql
 python:
-  - '3.4'
+  - '2.7'
```

Comment:
i believe so. http://docs.travis-ci.com/user/languages/python/#Choosing-Python-versions-to-test-against